script6.js

```diff
@@ -0,0 +1,27 @@
+/* 6.1 */
+
+let Cars = [];
+
+/* 6.2 */
+
+console.log("the value of my variable y will be: []");
```

Comment:
nit: small change, variable is not named `y`, should be `cars`

Reply:
Thanks I will correct.